tests/test_checks.py

```diff
@@ -236,8 +239,43 @@ def test_check_w007_invalid(self, mocked_guess_type):
             ],
         )
 
+    def test_debug_toolbar_installed_when_running_tests(self):
```

Comment:
I'm not sure why you're modifying the `dt_settings.get_config()` return value instead of using e.g. `with override_settings(...)`? I suspect that there's a good reason for it; if there is, please add a short docstring.

Reply:
If we were to use `override_settings`, we'd have to define the entire `DEBUG_TOOLBAR_CONFIG` dictionary. And I don't think there's a merge settings method.

Reply:
Let me know what you think of the new comment.

Reply:
Thanks! No further comments :-)